src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Copy/IntPtrArrayTests.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class IntPtrArrayTests
+    {
+        public static readonly object[][] ArrayData =
+        {
+            new object[] { GenerateIntPtrArray() }
+        };
+
+        public static IntPtr[] GenerateIntPtrArray()
+        {
+            IntPtr[] testArray = new IntPtr[10];
+            for (int i = 0; i < testArray.Length; i++)
+                testArray[i] = new IntPtr(i);
+            return testArray;
+        }
```

Comment:
If you wanted, this could also be simplified with LINQ, e.g.
```C#
public static readonly object[][] ArrayData =
{
    new object[] { Enumerable.Range(0, 10).Select(i => new IntPtr(i)).ToArray() }
};
```

Reply:
Thanks for the suggestion, fixed